change-notes/1.25/analysis-python.md

```diff
@@ -1,22 +1,9 @@
 # Improvements to Python analysis
 
-The following changes in version 1.25 affect Python analysis in all applications.
-
-## General improvements
-
-
-## New queries
-
-| **Query**                   | **Tags**  | **Purpose**                                                        |
-|-----------------------------|-----------|--------------------------------------------------------------------|
-
-
-## Changes to existing queries
-
-| **Query**                  | **Expected impact**    | **Change**                                                       |
-|----------------------------|------------------------|------------------------------------------------------------------|
-
-
-## Changes to libraries
-
 * Importing `semmle.python.web.HttpRequest` will no longer import `UntrustedStringKind` transitively. `UntrustedStringKind` is the most commonly used non-abstract subclass of `ExternalStringKind`. If not imported (by one mean or another), taint-tracking queries that concern `ExternalStringKind` will not produce any results. Please ensure such queries contain an explicit import (`import semmle.python.security.strings.Untrusted`).
+* Added model of taint sources for HTTP servers using `http.server`.
+* Added taint modeling of routed parameters in flask.
+* Improved modeling of builtin methods on strings for taint tracking.
+* Improved classification of test files.
+* New class `BoundMethodValue` exposing information about a bound method.
+* The query `py/command-line-injection` now recognizes command execution with the `fabric` and `invoke` Python libraries.
```

Comment:
I guess this should go into the above section "Changes to existing queries"

Reply:
If we followed the _old_ scheme of things, then yes, it should have gone into that table. I tried to follow the rest of the actual 1.25 change notes, and inlined this.

Reply:
This is also in line with what we hope do for change notes going forwards (and what Go is doing currently)